### ISSUE TITLE
[ENH] Probability of Default Model

### ISSUE THREAD

**Issue by jtoncelli** [luca-miniati/prosper-zero#15]
# Probability of Default Model

- [ ] Input variables: `amount_borrowed, prosper_rating, lender_yield, loan_status`
- [ ] NLL loss function
- [ ] Split data into training and validation
- [ ] Use TensorFlow.js to make model
